src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,11 +1,28 @@
 package core.basesyntax.service;
 
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_PASS_AGE = 18;
+    private static final int MIN_LENGTH_PASSWORD = 6;
+    private StorageDao storageData = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user == null
+                || user.getLogin() == null
+                || user.getAge() == null
+                || user.getPassword() == null) {
+            throw new NullPointerException("You have to fill all fields or check correct entry");
```

Comment:
```suggestion
            throw new RuntimeException("You have to fill all fields or check correct entry");
```

We should not throw NPE

Reply:
fixed